src/snowcli/utils.py

```diff
@@ -567,23 +566,7 @@ def convert_resource_details_to_dict(function_details: list[tuple]) -> dict:
 
 def conf_callback(ctx: typer.Context, param: typer.CallbackParam, value: str):
     if value:
-        try:
-            app_config = AppConfig().config
-
-            # Initialize the default map
-            ctx.default_map = ctx.default_map or {}
-            # if app_config has key 'default'
-            config_section = os.getenv("SNOWCLI_CONFIG_SECTION", "default")
-            if config_section in app_config:
-                ctx.default_map.update(
-                    app_config.get(config_section),
-                )  # type: ignore
-            if value in app_config:
-                # TODO: Merge the config dict into default_map
-                # type: ignore
-                ctx.default_map.update(app_config.get(value))
-        except Exception as ex:
-            raise typer.BadParameter(str(ex))
```

Comment:
Why you removed this code?

Reply:
Honestly, I don't know what this code is doing. I don't think we use the click context anywhere. WDYT?

Reply:
Right, let's remove it